src/worker/SandboxedWorker.js

```diff
@@ -42,7 +43,9 @@ class SandboxedWorker extends EventTarget {
 
     this.name = name
     this.iframe = document.createElement('iframe')
-    this.iframe.sandbox = 'allow-scripts'
+    if (!(iOS || isSafari)) {
```

Comment:
Let's add a comment here explaining why we don't do this, and linking to the bug as well.